actions/docs-verifier/src/RedirectionVerifier/RedirectionVerifier.csproj

```diff
@@ -4,11 +4,6 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="System.Collections.Immutable" Version="5.0.0" />
-    <PackageReference Include="System.Text.Json" Version="5.0.2" />
-  </ItemGroup>
-
```

Comment:
These were needed when the project targeted .NET Standard. They are redundant now.